builds/jenkins-build-sls-app/Jenkinsfile

```diff
@@ -0,0 +1,1062 @@
+      ]) {
+        def randomString = utilModule.generateRequestId();
+        def credsId = "jazz-${randomString}";
+        sh "aws configure set profile.${credsId}.region ${configLoader.AWS.REGION}"
+        sh "aws configure set profile.${credsId}.aws_access_key_id $AWS_ACCESS_KEY_ID"
+        sh "aws configure set profile.${credsId}.aws_secret_access_key $AWS_SECRET_ACCESS_KEY"
+
+        try {
+          sh "aws cloudformation validate-template --template-body file://.serverless/cloudformation-template-update-stack.json --region ${configLoader.AWS.REGION} > output.log"
+        } catch(ex) {
+          def outputLog = readFile('output.log').trim()
+          echo "template validation log $outputLog"
+          throw ex
+        }
+        def outputLog = readFile('output.log').trim()
+        echo "template validation log $outputLog"
+
+        def cft = readFile('./.serverless/cloudformation-template-update-stack.json').trim()
+        def cftJson = parseJson(cft)
+        def outstandingResources = whiteListModule.validate(cftJson)
+        if(!outstandingResources.isEmpty()) throw new Exception("Invalid CloudFormation template. The resource types not allowed: ${outstandingResources}")
+      }
+    }
+
+    stage("Deployment to ${env_key} environment") {
+      events.sendStartedEvent('DEPLOY_TO_AWS', 'Deployment started to staging AWS environment', context_map, environment_logical_id)
+      events.sendStartedEvent('UPDATE_ENVIRONMENT', "Environment status update event for ${env_key} deployment", environmentDeploymentMetadata.generateEnvironmentMap("deployment_started", environment_logical_id, null), environment_logical_id)
+
+      echo "starts Deployment to ${env_key} Env"
+      def lambdaARN = null
+      environmentDeploymentMetadata.setEnvironmentEndpoint(lambdaARN)
+
+      events.sendStartedEvent('UPDATE_ENVIRONMENT', "Environment status update event for ${env_key} deployment", environmentDeploymentMetadata.generateEnvironmentMap("deployment_started", environment_logical_id, null), environment_logical_id)
+
+      withCredentials([
+        [$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: configLoader.AWS_CREDENTIAL_ID, secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']
+      ]) {
+        try {
+          // initialize aws credentials
+          def randomString = utilModule.generateRequestId();
+          def credsId = "jazz-${randomString}";
+          sh "aws configure set profile.${credsId}.region ${configLoader.AWS.REGION}"
+          sh "aws configure set profile.${credsId}.aws_access_key_id $AWS_ACCESS_KEY_ID"
+          sh "aws configure set profile.${credsId}.aws_secret_access_key $AWS_SECRET_ACCESS_KEY"
+
+          // Generate serverless yml file with domain added in function name
+          echo "Generate deployment env with domain"
+
+          if (isScheduleEnabled) {
```

Comment:
Remove. Not applicable for services of type: `sls-app`.

Reply:
Resolving with https://github.com/tmobile/jazz/pull/829/commits/6b7312ff183a141dc3080f94e515915860c2f57c